plugins/Delay/DelayControlsDialog.cpp

```diff
@@ -22,10 +22,13 @@
  *
  */
 
-#include "delaycontrolsdialog.h"
-#include "delaycontrols.h"
+#include "DelayControlsDialog.h"
+#include "DelayControls.h"
 #include "embed.h"
 #include "TempoSyncKnob.h"
+#include "../Eq/EqFader.h"
```

Comment:
That could be a good idea, it could probably live in the Fader.h file, and the class needs renaming, maybe SelfUpdatingFader. Do you want me to proceed with this?